cmd/neofs-cli/modules/object.go

```diff
@@ -29,3 +130,368 @@ func init() {
 	// is called directly, e.g.:
 	// objectCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
 }
+
+func initSession(ctx context.Context) (*client.Client, *token.SessionToken, error) {
+	cli, err := getSDKClient()
+	if err != nil {
+		return nil, nil, fmt.Errorf("can't create client: %w", err)
+	}
+	tok, err := cli.CreateSession(ctx, math.MaxUint64)
+	if err != nil {
+		return nil, nil, fmt.Errorf("can't create session: %w", err)
+	}
+	return cli, tok, nil
+}
+
+func putObject(cmd *cobra.Command, args []string) error {
+	ownerID, err := getOwnerID()
+	if err != nil {
+		return err
+	}
+	cid, err := getCID(cmd)
+	if err != nil {
+		return err
+	}
+
+	filename := cmd.Flag("file").Value.String()
+	f, err := os.OpenFile(filename, os.O_RDONLY, os.ModePerm)
+	if err != nil {
+		return fmt.Errorf("can't open file '%s': %w", filename, err)
+	}
+
+	attrs, err := parseObjectAttrs(cmd)
+	if err != nil {
+		return fmt.Errorf("can't parse object attributes: %w", err)
+	}
+
+	obj := object.NewRaw()
+	obj.SetContainerID(cid)
+	obj.SetOwnerID(ownerID)
+	obj.SetAttributes(attrs...)
+
+	ctx := context.Background()
+	cli, tok, err := initSession(ctx)
+	if err != nil {
+		return err
+	}
+	oid, err := cli.PutObject(ctx,
+		new(client.PutObjectParams).
+			WithObject(obj.Object()).
+			WithPayloadReader(f),
+		client.WithSession(tok))
+	if err != nil {
+		return fmt.Errorf("can't put object: %w", err)
+	}
+
+	cmd.Printf("[%s] Object successfully stored\n", filename)
+	cmd.Printf("  ID: %s\n  CID: %s\n",
+		base58.Encode(oid.ToV2().GetValue()),
+		base58.Encode(cid.ToV2().GetValue()))
```

Comment:
With https://github.com/nspcc-dev/neofs-node/pull/98 you can just  `cmd.Printf("  ID: %s\n  CID: %s\n", oid, cid)`